Changes

```diff
@@ -8,6 +8,28 @@ The changes in each SiliconCompiler release version are described below. Commit
 version shown in (). Where applicable, the contributors that suggested a given
 feature are shown in [].
 
+SiliconCompiler 0.13.1 (2023-07-20)
+=========================================
+
+**Minor:**
+
+* Improved remote run reliability, including graceful time-outs when server not responding and thread-safety fixes.
+
+* Added ['option', 'libext'] support for Surelog, Verilator, and Icarus.
+
+* Removed dashboard support for Python 3.6.
+
+* Made aesthetic tweaks to dashboard.
+
+* Fixed bundled Surelog on pre-macOS 12.
+
+* Tools:
+
+  * yosys - fixed syn strategies.
+  * verilator - added multithreading, added FST trace format support.
+  * OpenROAD - bumped minimum version to better support IR drop analysis, added additional controls for abstract LEF generation, updated scripts to improve ORFS correspondence, fixed to use correct layers for parasitic estimation, improved/fixed PSM checking.
```

Comment:
PSM is IRDrop analysis, so you can remove that statement at the end
If you are referencing ORFS, I would include a link to their repo as well.